### PR TITLE
Ensure PhantomJS stderr is processed by line

### LINKED ISSUE #22: shoul ensure phantomsj stderr is processed by line
Hi,

In here 
https://github.com/kevva/screenshot-stream/blob/master/index.js#L63

You should ensure the stream is processed by lines, otherwise all the regex can break.

I faced a behavior where data event received chunks composed of multiple lines at once.
It happened when i started to send quiet some data. 
With small amount of exchanged data, I have not noticed that behavior.

fore reference,
I did that
```
var byline = require('byline');
byline(cp.stderr).on('data', function (data) {
    data = data.trim();
...
```

I rely on
https://github.com/jahewson/node-byline

### PR DESCRIPTION
**Pull request by maboiteaspam**
- It adds `byline` as a dependency, hope it s fine, otherwise i d like to get a suggestion.
-  ~~It also makes `xo` happier when validating `stream.js`, previously there was too many nested callbacks~~

-

Fixes #22.

### CODE REVIEW

File: stream.js [hunk 1]
```diff
@@ -92,11 +92,12 @@ page.open(opts.url, function (status) {
 	window.setTimeout(function () {
 		if (opts.hide) {
 			page.evaluate(function (els) {
-				els.forEach(function (el) {
+				var hideDomElement = function (el) {
```
Comment (shinnn) [inline stream.js]: Why?

Comment (maboiteaspam) [inline stream.js]: See in the last test run,

https://travis-ci.org/kevva/screenshot-stream/jobs/98161996

```
/home/travis/build/kevva/screenshot-stream/stream.js
  96:53  warning  Too many nested callbacks (5). Maximum allowed is 4  max-nested-callbacks
```

This warning is provided by `xo`.

This change fixes that warning.

Comment (sindresorhus) [inline stream.js]: Just ignore it. It's just a warning and I plan to change it.

**Comment by sindresorhus** [comment]
Can you also give the pull request a proper title?

**Comment by maboiteaspam** [comment]
open to comments if any : )

**Pull request closed without merging**